.github/workflows/install_ubuntu_dependencies_build.sh

```diff
@@ -1,4 +1,5 @@
 # Install required dependencies for Ubuntu systems
+pip install gcovr==6.0
```

Comment:
@shivaahir1508 since we have this, maybe we don't need lcov \ in dependencies. Please check that.

Reply:
@shivaahir1508 shouldn't this be pip3 ?

Reply:
Hi, I will do the changes.